velox/exec/Task.h

```diff
@@ -344,6 +344,11 @@ class Task : public std::enable_shared_from_this<Task> {
       uint32_t splitGroupId,
       const std::vector<core::PlanNodeId>& planNodeIds);
 
+  // Adds custom join bridges for all the specified plan node IDs.
```

Comment:
typo: "plan node IDs" -> "plan nodes" since the method takes plan nodes, not IDs.

Reply:
Fixed